external/@wordpress/blocks/index.js

```diff
@@ -1,32 +1,40 @@
 export { default as Editable } from './components/editable';
 
 /**
+ * Block settings keyed by block name.
+ *
+ * @var {Object} blocks
+ */
+const blocks = {};
+
+/**
  * Registers a block.
  *
- * @param  {string} namespace Block grouping unique to package or plugin
- * @param  {string} block     Block name
+ * @param  {string} name      Block name
  * @param  {Object} settings  Block settings
  */
-export function registerBlock( namespace, block, settings ) {
-
+export function registerBlock( name, settings ) {
+	if ( ! /^\w+\/\w+$/.test( name ) ) {
+		throw new Error( 'Use a namespace for your block name please. Example: namespace/block' );
+	}
+	blocks[ name ] = Object.assign( { name }, settings );
 }
 
 /**
  * Returns settings associated with a block.
  *
- * @param  {string}  namespace Block grouping unique to package or plugin
- * @param  {string}  block     Block name
- * @return {?Object}           Block settings
+ * @param  {string}  name Block name
+ * @return {?Object}      Block settings
  */
-export function getBlockSettings( namespace, block ) {
-
+export function getBlockSettings( name ) {
+	return blocks[ name ];
 }
 
 /**
  * Returns all registered blocks.
  *
- * @return {Object} Block settings keyed by block name
+ * @return {Array} Block settings
  */
 export function getBlocks() {
-
+	return Object.values( blocks );
```

Comment:
This will probably require pulling in [`babel-plugin-transform-runtime`](https://www.npmjs.com/package/babel-plugin-transform-runtime), otherwise will error on browsers where this is not yet implemented.